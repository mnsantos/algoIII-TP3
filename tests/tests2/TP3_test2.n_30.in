30 146
1 10
3 10
4 10
4 9
1 17
2 17
1 8
1 2
1 11
6 10
6 9
5 6
4 7
1 7
2 11
1 18
1 3
2 7
3 8
2 4
3 11
5 14
1 9
9 11
6 8
8 9
6 7
27 30
27 28
25 28
25 27
24 30
24 25
23 26
22 29
22 24
22 23
21 27
21 26
21 22
20 29
20 26
20 25
20 24
20 22
20 21
19 30
19 22
18 29
18 22
18 21
18 20
17 29
17 28
17 25
17 23
17 20
16 29
16 25
16 24
16 21
16 19
16 17
15 30
15 28
15 22
15 17
15 16
14 30
14 26
13 29
13 20
13 19
13 18
13 14
12 30
4 24
4 22
3 26
3 22
2 26
2 22
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
5 19
5 26
6 17
6 22
6 26
7 18
7 19
7 24
7 29
7 30
8 15
8 18
8 19
8 20
8 21
8 24
8 27
9 15
9 18
9 25
9 26
10 15
10 16
10 17
10 20
10 23
10 26
10 28
11 14
11 17
11 18
11 19
11 22
11 27
11 30
12 13
12 16
12 17
12 29
1 15
6 15
3 4
1 4
4 11
1 13
10 11
1 6
1 12
1 14
7 12
1 5
1 16
0
