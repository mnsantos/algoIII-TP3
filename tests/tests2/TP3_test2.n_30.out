55 4 1 10 11 17
